### DISCUSSION TITLE
Can't Remove EPG Lineup

### DISCUSSION THREAD

**Discussion by jchiso** [OpenSageTV/Community#105] [Guide/EPG]
I've recently had a SchedulesDirect EPG lineup retirement, so I've been attempting to update to a new lineup.  The update process has produced a myriad of confusing results. I've been able to update one (of four) Sage servers to this point, but the others have experienced insurmountable issues.
One issue I'm experiencing now is that when I attempt to 'Remove' an EPG lineup within Sage and 'Add a New Lineup', the old lineup does not get replaced and/or the new lineup is not selectable.

Is there a way (perhaps by editing the .properties file) to remove the old, retired EPG lineup?

**Comment by KryptoNyte-Sage** [comment]
Maybe someone else can comment on this, but I don't believe I've ever deleted a Schedules Direct Lineup from within SageTV.  When I log in to my Schedules Direct account on their own website, they claim that the JSON lineups are all handled within the [SageTV] application, and although there is no 'Add Lineup' button in the Schedules Direct account on their website, there is a 'Delete' button beside each lineup.

**Reply by jchiso** [reply]
I seem to have resolved my issues, though they varied from server to server.  Ultimately, what I think worked was establishing the lineups on the SchedulesDirect site and then selecting the new lineup from the configuration screens withing Sage.  The biggest issue was that in a few cases (and a few times) the name of the new lineup would not appear as an option, or the name shown within Sage did not match the lineup name in SchedulesDirect.

I should also mention that editing the channel list on the SchedulesDirect site did not produce a comparable, active channel list within Sage.
